specs/lib/bakeAmbientOcclusionSpec.js

```diff
@@ -0,0 +1,501 @@
+                "byteLength": 4 * 2 * 4,
+                "target": 34962
+            }
+        },
+        "buffers": {
+            "buffer_0": {
+                "uri": "data:",
+                "byteLength": indices.length * 2 + (positions.length + normals.length + uvs.length) * 4,
+                "extras": {
+                    "_pipeline": {
+                        "source": dataBuffer
+                    }
+                }
+            }
+        },
+        "scene": "defaultScene",
+        "scenes": {
+            "defaultScene": {
+                "nodes": [
+                    "squareNode"
+                ]
+            }
+        },
+        "nodes": {
+            "squareNode": {
+                "children": [],
+                "matrix": [
+                    2, 0, 0, 0,
+                    0, 2, 0, 0,
+                    0, 0, 2, 0,
+                    0, 0, 0, 1
+                ],
+                "meshes": [
+                    "mesh_square"
+                ],
+                "name": "square",
+                "extras": {
+                    "_pipeline": {}
+                }
+            }
+        },
+        "meshes": {
+            "mesh_square": {
+                "name": "square",
+                "primitives": [
+                    {
+                        "attributes": {
+                            "POSITION": "accessor_position",
+                            "NORMAL": "accessor_normal",
+                            "TEXCOORD_0": "accessor_uv"
+                        },
+                        "indices": "accessor_index",
+                        "extras": {
+                            "_pipeline": {}
+                        }
+                    }
+                ]
+            }
+        }
+    };
+
+    beforeAll(function(done) {
+        var loadModel2 = function () {
+            fs.readFile(boxOverGroundGltfPath, function (err, data) {
+                if (err) {
+                    throw err;
+                }
+                else {
+                    boxOverGroundGltf = JSON.parse(data);
+                    addPipelineExtras(boxOverGroundGltf);
+                    loadGltfUris(boxOverGroundGltf, loadGltfUriOptions, function (err, gltf) {
+                        if (err) {
+                            throw err;
+                        }
+                        done();
+                    });
+                }
+            });
+        };
+
+        fs.readFile(boxGltfPath, function(err, data) {
+            if (err) {
+                throw err;
+            }
+            else {
+                boxGltf = JSON.parse(data);
+                addPipelineExtras(boxGltf);
+                loadGltfUris(boxGltf, loadGltfUriOptions, function(err, gltf) {
+                    if (err) {
+                        throw err;
+                    }
+                    loadModel2();
+                });
+            }
+        });
```

Comment:
Hm just noticed, this area could be simplified by using `readGltf` if that works.